documentation/book/cluster-operator.adoc

```diff
@@ -261,6 +261,13 @@ below.
 }
 ----
 
+Selected options have default values:
+
+* `controlled.shutdown.enable` with default value `true`
+* `auto.leader.rebalance.enable` with default value `true`
```

Comment:
AFAICS from the upstream kafka docs, both of these default to true anyway. In which case, there's no need for this PR. Am I missing something?

Reply:
Hmm, I though they default to false. But you seem to be right. No need for this PR then :-(.